datahub-web-react/src/app/home/AcrylDemoBanner.tsx

```diff
@@ -52,7 +52,7 @@ export default function AcrylDemoBanner() {
                     >
                         Schedule a demo
                     </StyledLink>{' '}
-                    of Acryl DataHub to see the advanced features that take it to the next level or purchase Acryl Cloud
+                    of DataHub Cloud to see the advanced features that take it to the next level or purchase DataHub Cloud
```

Comment:
I wonder if we need to say "or purchase DataHub Cloud"

Reply:
hmm make sense - but I would prefer not to include the change in this PR, maybe we should make another one